seL4-platforms/platforms.py

```diff
@@ -109,6 +110,7 @@ def __repr__(self):
             f"    req: {self.req}",
             f"    disabled: {self.disabled}",
             f"    no_hw_build: {self.no_hw_build}",
+            f"    extra_settings: {self.extra_settings}",
```

Comment:
This will print
```
extra_settings: {'foo': 'foo1', 'bar': 'bar1'}
```
or
```
extra_settings: {}
```
but I assume we want YAML format:
```
extra_settings:
  foo: foo1
   bar: bar1
```

Reply:
This is still blocking the merge, unless we don't need YAML output here?

Reply:
Seems we don't need YAML.